src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -357,15 +356,20 @@ public function initialized($id)
      */
     public function getServiceIds()
     {
-        $ids = array();
+        $ids = array_keys($this->services + $this->methodMap);
```

Comment:
should be the other way around btw: `$this->methodMap + $this->services` to mimic current ordering (at prevent changing test cases, isn't it)?